doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -108,7 +109,12 @@ Then you need to add the ebpf flags to configure ::
  sudo mkdir /etc/suricata/ebpf/
 
 The ``clang`` compiler is needed if you want to build eBPF files as the build
-is done via a specific eBPF backend available only in llvm/clang suite.
+is done via a specific eBPF backend available only in llvm/clang suite. If you
```

Comment:
configure will pick clang as eBPF compiler if CC is gcc and `--enable-ebpf --enable-ebpf-build` is set. So documentation is not optimal but I think it is good enough.